react-design/my-app/src/Indicators.tsx

```diff
@@ -0,0 +1,22 @@
+import { SetStateAction } from 'react';
+
+type Props = {
+  count: number;
+  index: number;
+  setIndex: React.Dispatch<SetStateAction<number>>;
```

Comment:
It's best to declare these as actual functions instead of React.Dispatch. Like this:
`setIndex: (index: number) => void`